packages/tracking/package.json

```diff
@@ -28,10 +28,11 @@
     "lint:typescript": "../../scripts/check-typescript"
   },
   "devDependencies": {
-    "prop-types": "^15.8.1"
+    "prop-types": "^15.8.1",
+    "nebenan-helpers": "^6.0.0"
```

Comment:
why do we need this in devDependencies?

Reply:
Added it here, because VSCode was shouting at me that this is missing in deps :)

Reply:
But its in the root package.json.... Webstorm doesnt complain :-(

Reply:
Hmm true! Will get rid of it again!